test/Microsoft.ApplicationInsights.AspNet.Tests/TelemetryInitializers/OperationNameTelemetryInitializerTests.cs

```diff
@@ -72,16 +71,107 @@ public void InitializeSetsTelemetryOperationNameToMethodAndPath()
         public void InitializeSetsRequestNameToMethodAndPath()
         {
             var telemetry = new RequestTelemetry();
-            var request = new DefaultHttpContext().Request;
-            request.Method = "GET";
-            request.Path = new Microsoft.AspNet.Http.PathString("/Test");
-            var ac = new HttpContextAccessor() { HttpContext = request.HttpContext };
-            ac.HttpContext.RequestServices = new TestServiceProvider(new List<object>() { new RequestTelemetry() });
-            var initializer = new OperationNameTelemetryInitializer(ac);
+            var contextAccessor = HttpContextAccessorHelper.CreateHttpContextAccessor(telemetry, null);
+
+            var initializer = new OperationNameTelemetryInitializer(contextAccessor);
 
             initializer.Initialize(telemetry);
 
             Assert.Equal("GET /Test", telemetry.Name);
         }
+
+        [Fact]
+        public void InitializeSetsTelemetryOperationNameToControllerFromActionContext()
+        {
+            var actionContext = new ActionContext();
+            actionContext.RouteData = new Microsoft.AspNet.Routing.RouteData();
```

Comment:
Why do we use the full namespace here instead of a using directive at the top of the file?